### PR TITLE
fix: `semantic-release` require node >= 18

### PR DESCRIPTION
**Pull request by jamacku**
See https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md for more details and solutions.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ jobs:
           git push --force https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git HEAD:refs/heads/v3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - run: npx semantic-release
+      # Workaround for [semantic-release]: node version >=18 is required. Found v16.19.0.
+      # See https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md for more details and solutions.
+      - run: npx -p node@v18-lts -c "npx semantic-release"
```
Comment (gr2m) [inline .github/workflows/release.yml:28]: I would just update line 14 to this instead

```yml
          node-version: "lts/*"
```

Comment (jamacku) [reply .github/workflows/release.yml:28]: Good point. I didn't know about this option. I'll update PR.

Comment (jamacku) [reply .github/workflows/release.yml]: Done

**Review by stefanbuck — APPROVED** [review]

**Comment by stefanbuck** [comment]
Thanks for the fix

**Pull request merged**